lib/configuru/version.rb

```diff
@@ -1,3 +1,3 @@
 module Configuru
-  VERSION = "0.1.2"
+  VERSION = '0.1.2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.